front-end/package.json

```diff
@@ -24,15 +33,62 @@
     "react-player": "^2.15.1",
     "react-resize-detector": "^10.0.1",
     "react-router-dom": "^6.22.3",
+    "tree-kill": "^1.2.2",
     "vite": "^5.4.3",
     "vite-plugin-commonjs": "^0.10.1",
     "vitest": "^2.1.1"
   },
   "scripts": {
     "start": "vite",
-    "build": "tsc && vite build",
+    "build:frontend": "tsc && vite build",
+    "build:electron": "electron-builder",
     "serve": "vite preview",
-    "lint": "npx eslint . --fix"
+    "lint": "npx eslint . --fix",
+    "electron": "electron src/electron/main.js --trace-warnings",
+    "dev": "concurrently \"npm run start\" \"npm run electron\""
+  },
+  "build": {
+    "appId": "com.dataviewer.mcmaster",
+    "asar": true,
+    "files": [
+      "src/electron/main.js",
+      "build/**/*"
+    ],
+    "extraResources": [
+    {
+      "from": "backend",
+      "to": "backend",
+      "filter": [
+        "**/*"
+      ]
+    }
+  ],
+    "win": {
+      "icon": "public/bajacar.ico",
+      "target": [
+        {
+          "target": "nsis",
+          "arch": [
+            "x64"
+          ]
+        }
+      ],
+      "artifactName": "${productName}-v${version}-win.exe"
+    },
+    "nsis": {
+      "oneClick": false,
+      "allowElevation": true,
+      "allowToChangeInstallationDirectory": true,
+      "installerIcon": "public/bajacar.ico",
+      "uninstallerIcon": "public/bajacar.ico",
+      "installerHeaderIcon": "public/bajacar.ico",
+      "createDesktopShortcut": true,
+      "createStartMenuShortcut": true,
```

Comment:
This is fun

Reply:
does anything need to be changed?

Reply:
updated to not create start menu shortcut